buildpack/telemetry/appdynamics.py

```diff
@@ -18,6 +18,63 @@
     "machine-agent",
 )
 
+APPDYNAMICS_NODE_NUMBER = os.getenv("CF_INSTANCE_INDEX", default=0)
+APPDYNAMICS_TIER = util.get_vcap_data()["application_name"]
+
+# Number suffix will be added to AppDynamics Node Name
+# The key will be used in the function '_filter_default_env'
+APPDYNAMICS_NODE_NAME_KEY = "APPDYNAMICS_AGENT_NODE_NAME"
+
+APPDYNAMICS_DEFAULT_ENV_VARS = {
+    "APPDYNAMICS_AGENT_APPLICATION_NAME": util.get_app_from_domain(),
+    APPDYNAMICS_NODE_NAME_KEY: "node",  # This is 'base' name without suffix
+    "APPDYNAMICS_AGENT_TIER_NAME": APPDYNAMICS_TIER,
+    "APPDYNAMICS_CONTROLLER_PORT": "443",
+    "APPDYNAMICS_CONTROLLER_SSL_ENABLED": "true",
+}
+
+
+def _filter_default_env():
```

Comment:
Falling back to a default value is already provided by `os.getenv`. Do we need this function ?

Reply:
Good point, changed